docs/reference/commandline/cp.md

```diff
@@ -22,15 +22,15 @@ The `docker cp` utility copies the contents of `SRC_PATH` to the `DEST_PATH`.
 You can copy from the container's file system to the local machine or the
 reverse, from the local filesystem to the container. If `-` is specified for
 either the `SRC_PATH` or `DEST_PATH`, you can also stream a tar archive from
-`STDIN` or to `STDOUT`. The `CONTAINER` can be a running or stopped container.
-The `SRC_PATH` or `DEST_PATH` can be a file or directory.
+`STDIN` or to `STDOUT`. The `CONTAINER` is a running or stopped container.
+The `SRC_PATH` or `DEST_PATH` is a file or directory.
```

Comment:
hm, not sure about this; "can be" was added to illustrate that the container doesn't have to be running, changing to "is" doesn't add much, I think